website/other/bezier-rs-demos/src/App.vue

```diff
@@ -315,6 +315,75 @@ export default defineComponent({
 						},
 					},
 				},
+				{
+					name: "Arcs",
+					callback: (bezier: WasmBezierInstance, options: Record<string, number>): string => bezier.arcs(options.error, options.max_iterations, options.strategy),
+					exampleOptions: {
+						[BezierCurveType.Quadratic]: {
+							customPoints: [
+								[50, 50],
+								[85, 65],
+								[100, 100],
+							],
+							sliderOptions: [
```

Comment:
Abstract out the common sliders, and then use it for both the quadratic and cubic options to avoid repetition